openid-sharedsignals-framework-1_0.md

```diff
@@ -240,10 +266,20 @@ Subject Principals are identified by Subject Members defined below.
 
 # Subject Members in SSF Events {#subject-ids}
 
-A member of type Subject in an SSF event MAY have any claim name. Each Subject Member MUST
-refer to exactly one Subject Principal.
+## Subject Members {#subject-members}
+A Subject Member of a SSF event describes a subject of the event. A top-level claim named `sub_id` SHOULD be used to describe the primary Subject Principal of the event.
+
+### Existing CAEP and RISC Events
+Event types already defiend in the CAEP ({{CAEP}}) and RISC ({{RISC}}) drafts MAY use a `subject` field within the `events` claim of the SSF eent to describe the primary Subject Principal of the event. SSF Transmitters SHOULD include the top-level `sub_id` claim even for these existing event types.
+
+### New Event Types
+New event types MUST use the top-level `sub_id` claim and MUST NOT use the `subject` field in the `events` claim to describe the primary Subject Principal.
 
-A Subject may be a "simple subject" or a "complex subject".
+### Additional Subject Members
```

Comment:
Could you kindly indicate the location where the "Additional Subject Members" would be listed?

Reply:
I think it would be totally up to the authors of the new event types. For instance, I might create a "got married" event that looks like
```
{
  "date": "2009-11-23",
  "location": "New York, NY",
  "subject_1": "Avery Apple",
  "subject_2": "Blair Bonavue"
}
```

This section is just saying that it is perfectly fine for me to create new event types that include subject information, and that I am not restricted in the naming of those fields.